### PR TITLE
New Installation article

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description

This a new article related only to the Flyve MDM version 2.0.0-dev
It keeps:
* the glpi configuration required to be able to access the wizard, with images updated
* the compatibility matrix

References to the wizard implementation

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### References

Reference #287

### CODE REVIEW

File: _posts/static/2018-02-01-installation-wizard.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+### Compatibility matrix
+
+<table>
+    <tr>
+        <td style="width:150px">GLPI</td>
+        <td style="width:100px">9.1.1</td>
+        <td style="width:100px">9.1.2</td>
+        <td style="width:100px">9.1.3</td>
+        <td style="width:100px">9.2.0</td>
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: 9.2.1 required for the development version and the incoming release

The  text above the chart may need an update if you want to refer to the next release.

File: _posts/static/2018-02-01-installation-wizard.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <td>9.1+1.0</td>
+        <td>9.2+1.0</td>
+    </tr>
+</table>
+
+## Early Steps
+
+* Download our specific version of GLPI 9.2.x (please refer to its documentation to install)
+* Go to the glpi folder and **run composer install --no-dev**
+* Download FusionInventory 9.2+1.0 for GLPI and put it in GLPI/plugins/
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: GLPI in lower case

File: _posts/static/2018-02-01-installation-wizard.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+<img src="{{ '/images/glpi-tree-structure.png' | absolute_url }}" alt="GLPI Tree Insfrastructure">
+
+* Go in the directory glpi/plugins/flyvemdm
+* Run **composer install --no-dev**
+
+## Configuration of GLPI
+
+These steps are mandatory in order to access the plugin configuration.
+
+### Notifications
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: Notification configuration may also be achieved with the wizard. Maybe you may add a note about that.

File: _posts/static/2018-02-01-installation-wizard.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  * Enable followup
+  * Enable followups via email
+
+<img src="{{ '/images/step-notifications.png' | absolute_url }}" alt="Enable Email Notifications">
+
+Click on **Email followups configuration** and setup the form depending on your requirements to send emails. This step is available from the wizard.
+
+<img src="{{ '/images/step-email-notifications.png' | absolute_url }}" alt="Email Notification Settings">
+
+### Enabling the Rest API
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: API configuration may also be achieved with the wizard. Maybe you may add a note about that.

File: _posts/static/2018-02-01-installation-wizard.md (hunk 2 above)
Comment (Naylin15) [reply _posts/static/2018-02-01-installation-wizard.md]: Changing...

File: _posts/static/2018-02-01-installation-wizard.md (hunk 3 above)
Comment (Naylin15) [reply _posts/static/2018-02-01-installation-wizard.md]: Hi, @btry 

I already added at the end with the email settings that this step is available from the wizard

File: _posts/static/2018-02-01-installation-wizard.md (hunk 4 above)
Comment (Naylin15) [reply _posts/static/2018-02-01-installation-wizard.md]: Adding...

**Pull request merged**